modules/calendar/add_edit_event.php

```diff
@@ -197,16 +197,37 @@ function setEventDate($start_date, $recurrence){
  function refreshCalendar() {
     echo "<script type='text/javascript'>";
       echo "var docRoot = top.$('#mainBox');
-            //Calendar frame opened during initial loading of EHR
-            var calFrame1 = docRoot.find('iframe[name=" . "lst" . "]');
-            var calSrc1 = calFrame1.attr('src');
-            if (calSrc1 !== undefined) {
-                calFrame1.attr('src', calSrc1);
+            //get references to all possible frames for calendar
+            var leftFrame = docRoot.find('iframe[name=" . "lst" . "]'); //initial loading of EHR
```

Comment:
These loadings are with the standard default, and are configurable.  Trying to determine if that is an issue at all.

Reply:
@aethelwulffe Yes, the calendar could be loaded into either frame (left or right) or not loaded at all upon logging in. Is that what you meant? I've taken cases for both in the function.

Reply:
So long as we keep the standard that there will never be THREE default tabs (good idea in my opinion), and so long as we do another test with Terry's "Single Page Application" global setting, I think we are all on the same iFrame.  Good job.